packages/ui-react/src/components/Card/Card.module.scss

```diff
@@ -103,12 +103,6 @@
     left: 0;
     width: 100%;
     height: 100%;
-    opacity: 0;
```

Comment:
Does it also disable opacity for desktop devices?

Reply:
This is moved to the `Shelf.module.scss` file because we don't want to limit card re-renders while sliding